<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SgProgWidget</class>
 <widget class="QWidget" name="SgProgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SgProgWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>有效相位数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="countPhaseLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>开始相位：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="startPhaseComboBox">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>控制方式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ctrlTypeComboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>绿灯间隔：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>相序：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_squence"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>周期时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="cycleSpinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>偏移：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="offSetSpinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>切换点：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="switchPointSpinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="sgProgTableWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>信号灯组</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>相位类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>协调相位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最小绿</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最大绿1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最大绿2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>延长绿</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>信号序列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>图形设置</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>红灯1结束</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/red</normaloff>:/lights/red</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>红闪1结束</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/flashing_red</normaloff>:/lights/flashing_red</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>绿灯1结束</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/green</normaloff>:/lights/green</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>红灯2结束</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/red</normaloff>:/lights/red</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>红闪2结束</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/flashing_red</normaloff>:/lights/flashing_red</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>绿灯2结束</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/green</normaloff>:/lights/green</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>红/黄</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/red_amber</normaloff>:/lights/red_amber</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>黄灯</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/amber</normaloff>:/lights/amber</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>红闪</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/lights/flashing_red</normaloff>:/lights/flashing_red</iconset>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
